src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -935,25 +938,31 @@ public void testInspectContainerWithSecurityOpts() throws Exception {
 
   @Test
   public void testContainerWithHostConfig() throws Exception {
```

Comment:
Maybe we should add an extra test specifically for container creation where the HostConfig is in the ContainerConfig so the assumeTrue doesn't ignore this test for docker API versions less than 1.18.